New version of multi active satellite macro

DVTimWilson opened the pull request:
[AB#1989](https://dev.azure.com/data-vault/dbtvault/_workitems/edit/1989/)
New version of multi active satellite macro for Snowflake, MSSQL and BigQuery.
Also additional test coverage.
Documentation update here: Datavault-UK/dbtvault-docs#14

DVAlexHiggs commented:
This looks really nice. So good to see this simplified so much!

Excited to get this to our users, thanks for your work 💯

DVTimWilson commented:
Automatic test failure due to timeout, possibly because 'mas_one_cdk_base_sats_rank_mat' was specified twice in separate jobs?
Pushed changes for this, and also added a missing test to the 1cdk job spec and removed a stray comma.

DVAlexHiggs commented:
Thanks Tim. Just one observation, why do we have feature files called `x_updated.feature` this is not very clear, and it also leads me to believe there may be duplication in tests between these and the same feature files without the `_updated` can you confirm?

DVTimWilson commented:
> Thanks Tim. Just one observation, why do we have feature files called `x_updated.feature`? this is not very clear, and it also leads me to believe there may be duplication in tests between these and the same feature files without the `_updated` can you confirm?

Doh! things left behind and all that. We'll get this sorted out shortly.

DVAlexHiggs commented:
I can now merge this, however the file is still called "_updated" please change this once merged.

DVAlexHiggs reviewed (approved):
Thanks, I can now merge this, however the file is still called "_updated" please change this once merged.

DVTimWilson commented:
> Thanks, I can now merge this, however the file is still called "_updated" please change this once merged.

Hmmm, that file is definitely renamed for me.

DVAlexHiggs commented:
test/features/ma_sats/bigquery/bigquery_mas_one_cdk_0_base_updated.feature
